### PR TITLE
feat(angular): add support for incremental builds to the webpack-server executor

### PR DESCRIPTION
**Pull request by leosvelperez**
## Current Behavior

The `@nrwl/angular:webpack-server` executor doesn't support incremental builds.

## Expected Behavior

The `@nrwl/angular:webpack-server` executor supports incremental builds.

## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/angular/src/builders/webpack-server/lib/configure-buildable-libs-support.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  );
+
+  // We can't just pass the tsconfig path in memory to the angular builder
+  // function because we can't pass the build target options to it, the build
+  // targets options will be retrieved from the project configuration.
+  // Therefore, we overwrite the tsconfig path that comes in the webpack
+  // configuration for the AngularWebpackPlugin to use the temporary tsconfig.
+  const angularWebpackPlugin = baseWebpackConfig.plugins.find(
+    (plugin) => plugin.constructor.name === AngularWebpackPlugin.name
+  ) as unknown as AngularWebpackPlugin;
```
Comment (Coly010) [inline packages/angular/src/builders/webpack-server/lib/configure-buildable-libs-support.ts:46]: This double cast is interesting. What's the reasoning behind this?

**Review by Coly010 — APPROVED** [review]
LGTM! :tada:

File: packages/angular/src/builders/webpack-server/lib/configure-buildable-libs-support.ts (hunk 1 above)
Comment (leosvelperez) [reply packages/angular/src/builders/webpack-server/lib/configure-buildable-libs-support.ts:46]: The `baseWebpackConfig.plugins` is of type `WebpackPluginInstance` which can't be casted directly to `AngularWebpackPlugin`.

**Review by Coly010 — APPROVED** [review]
LGTM! 🎉

**Pull request merged**